workflow/main.rb

```diff
@@ -22,6 +22,15 @@ def emoji_item(emoji)
   }
 end
 
+def is_image_only(emoji)
+  if emoji.image_filename == "#{emoji.name}.png"
```

Comment:
Two improvements to suggest:

1. You can use `String#end_with?` to test for a `.png` ending.
2. You can simply return `emoji.image_filename.end_with?(".png")`, rather than setting and returning a variable.